lib/Dialect/Ibis/IbisOps.cpp

```diff
@@ -141,6 +141,143 @@ LogicalResult ReturnOp::verify() {
 
   return success();
 }
+
+//===----------------------------------------------------------------------===//
+// InstanceOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult InstanceOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
+  auto targetClass = getClass(&symbolTable);
+  if (!targetClass)
+    return emitOpError() << "'" << getClassName() << "' does not exist";
+
+  return success();
+}
+
+//===----------------------------------------------------------------------===//
+// PortReadOp
+//===----------------------------------------------------------------------===//
+
+template <typename TSrcOp, typename TTargetOp>
```

Comment:
Could you note (in a comment) what you expect TSrcOp and TTargetOp to be (e.g. read/write, port op)?

Reply:
In general, I think this helps readability given that C++ still doesn't have concepts.